pyannote/audio/interactive/v03.py

```diff
@@ -0,0 +1,193 @@
+from copy import deepcopy
+from pathlib import Path
+from typing import Any, Dict, Iterable, List, Optional, Union
+
+import prodigy
+from answers2file import answers2file
+from prodigy.components.db import connect
+from prodigy.components.loaders import Audio
+from prodigy.core import recipe
+from prodigy.util import log
+from utils import SAMPLE_RATE, chunks, normalize, to_audio_spans, to_base64
+
+from pyannote.audio.core.io import Audio as AudioFile
+from pyannote.audio.pipelines import VoiceActivityDetection
+from pyannote.core import Annotation, Segment
+from pyannote.pipeline import Optimizer
+
+# IMPORT POUR SAD
+
+
+db = connect("sqlite", {})
+
+# Global Model
+pipeline = VoiceActivityDetection(segmentation="pyannote/segmentation", step=0.5)
+HYPER_PARAMETERS = {
+    "onset": 0.5,
+    "offset": 0.5,
+    "min_duration_on": 0.0,
+    "min_duration_off": 0.0,
+}
+pipeline.instantiate(HYPER_PARAMETERS)
+
+
+def update(answers):
+    prodigy.log("---- UPDATE ----")
+    annotations = db.get_dataset("asupprimer")
+    validation_files = answers2file(annotations)
+    optimizer = Optimizer(pipeline)
+    bp = optimizer.tune(
+        validation_files,
+        n_iterations=2,
+        show_progress=False,
+        warm_start=pipeline.parameters(instantiated=True),
+    )
+    pipeline.instantiate(bp["params"])
+    prodigy.log(bp)
+    prodigy.log("---- END UPDATE ----")
```

Comment:
Can you remove `update` completely for now?
As discussed the fine-tuning part will most likely take place in a separate script. 
So let's make this Pull Request (PR) simpler and focus on solving #765 and #766.